content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 'Get Started: Using and deploying models'
+description:
+  'Easily download your models from the model registry. Set up your CICD
+  pipelines to be trigger by model registry actions (such as assigning model
+  stages) and deploy models directly form the model registry.'
+---
+
+# Get Started: Using and Deploying Models
+
+In this chapter, we will learn how to download and use models and use the model
+registry to trigger automated CICD model workflows.
+
+## Downloading models
+
+It is useful to download model artifacts for example for local testing or for
+use in CICD workflows. With models versioned by DVC this can be done easily by
+using the `dvc get` CLI command as follows
+
+```bash
+dvc get https://<path-to-my-git-forge-account>/example-get-started-experiments models/model.pth --rev v1.0.0
+```
+
+where you just need to replace `<path-to-my-git-forge-account>` with your
+GitHub/GitLab/Bitbucket account path. This will download the model version
+v1.0.0 to the location from where you are calling the `dvc get` command.
+
+If you're using Studio this is even easier. Go to the Models dashboard, click on
+the three dots next to the name of your model and select "Download model file".
+Then you can select the model version you're interested in and Studio will
+generate the correct call of the `dvc get` command for you. You then just need
+to copy it and run it where you want the model to be downloaded. You can see all
+the steps here:
+
+![Downloading models](/img/mr-download-model.gif)
+
+## Connecting model registry actions to your CICD
+
+As we [noted](/docs/start/model-management/model-registry#GTO-tip) in the model
+registry chapter, all DVC model registry actions are captured in your git
+repository as git tags with a specific format.
+
+This also means that we can create CICD actions in our git repository which will
+be triggered whenever.
+
+In the following, we will have a look at an example CICD workflow on GitHub
+which runs whenever we assign a version of our model to the "test" stage in the
+model registry. The workflow will check whether the model artifact size is less
+than 50 MB and fail if it isn't. To motivate the example, we can imagine that we
+are deploying models to edge devices where storage comes at a premium.
```

Comment:
Not that strong of an opinion, but generally we have avoided artificial scenarios like this, and I'm not sure the check adds that much here. Maybe I miss the point you are going for here.

Reply:
The main point is to create a workflow which you can run and test without having to set up anything extra like AWS credentials, roles, permissions...But yeah, the scenario is artificial. I couldn't think about anything less artificial, but perhaps we could just embrace the fact that it is not a real example and just have the workflow do something like `echo "Here is where the model would be deployed to your deployment environment"`

Reply:
Added https://github.com/iterative/example-repos-dev/issues/272 as a follow up. I think at least starting with building and pushing a docker image would be a good simple start. Until then, I would rather embrace that it's not a real example for now like you suggested.